src/Log/Engine/BaseLog.php

```diff
@@ -159,4 +159,14 @@ protected function _format(string $message, array $context = []): string
 
         return str_replace(array_keys($replacements), $replacements, $message);
     }
+
+    /**
+     * Get the date.
+     *
+     * @return string
+     */
+    protected function _date()
```

Comment:
I would prefer a better name and a more complete description.

Reply:
It should especially note that one has to implement the required format option, or the option should be moved into the base log.